packages/env-variables/src/common/base-env-variables-protocol.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc. and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+export const baseEnvVariablesPath = '/services/envs';
+
+export const IBaseEnvVariablesServer = Symbol("IBaseEnvVariablesServer");
+export interface IBaseEnvVariablesServer {
+
+    getEnvList(): Promise<EnvVariable[]>
+
+    getEnvValueByKey(key: string): Promise<string>
```

Comment:
Would you consider to simplify interface?
```ts
export interface EnvVariableServer {
    getVariables(): Promise<EnvVariable[]>;
    getValue(key: string): Promise<string>;
}
```

Could you clarify why you need `Base` prefix?